Source/Cells/BrickCell.swift

```diff
@@ -117,8 +117,9 @@ open class BaseBrickCell: UICollectionViewCell {
         brickBackgroundView?.frame = self.bounds
 
         if _brick != nil && frame.width == requestedSize.width {
-            self.layoutIfNeeded() // This layoutIfNeeded is added to make sure that the subviews are laid out correctly
-            self.framesDidLayout()
+            DispatchQueue.main.async { // Allows to return for layout pass
```

Comment:
Wouldn't we already on the main queue

Reply:
I ended up undoing this change. Correct, it still on the main thread. My thought here was doing this puts self.frameDidLayout() on it's own stack, but actually it's fine the way it is.